### PR TITLE
add in latent for assualt instances

### PR DESCRIPTION
**Pull request by KnowOne134**

### CODE REVIEW

File: src/map/entities/battleentity.cpp [hunk 1]
```diff
@@ -111,6 +111,14 @@ bool CBattleEntity::isInDynamis()
     return false;
 }
 
+bool CBattleEntity::isInAssault()
+{
+    if (loc.zone != nullptr) {
```
Comment (takhlaq) [inline src/map/entities/battleentity.cpp]: curly brace on new line

File: src/map/entities/battleentity.cpp [hunk 2]
```diff
@@ -111,6 +111,14 @@ bool CBattleEntity::isInDynamis()
...
+        return (loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= 28 && loc.zone->GetRegionID() <= 32));
```
Comment (takhlaq) [inline src/map/entities/battleentity.cpp]: unneeded outer parentheses

File: src/map/entities/battleentity.cpp [hunk 3]
```diff
@@ -111,6 +111,15 @@ bool CBattleEntity::isInDynamis()
     return false;
 }
 
+bool CBattleEntity::isInAssault()
+{
+    if (loc.zone != nullptr) 
+    {
+        return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= 28 && loc.zone->GetRegionID() <= 32);
```
Comment (takhlaq) [inline src/map/entities/battleentity.cpp]: region enums

Comment (KnowOne134) [reply src/map/entities/battleentity.cpp]: Sorry dont know what you mean

Comment (KnowOne134) [reply src/map/entities/battleentity.cpp]: return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= 28 && loc.zone->GetRegionID() <= 32);

to

 return loc.zone->GetType() == ZONETYPE_DUNGEON_INSTANCED && (loc.zone->GetRegionID() >= REGION_WEST_AHT_URHGAN && loc.zone->GetRegionID() <= REGION_ALZADAAL);

is that what your looking for?

**Pull request merged**